test/storage/2019-07-07/azblob/zz_generated_pageblob.go

```diff
@@ -52,17 +51,17 @@ func (client *pageBlobClient) clearPagesCreateRequest(ctx context.Context, conte
 	req.URL.RawQuery = query.Encode()
 	req.Header.Set("x-ms-page-write", "clear")
 	req.Header.Set("Content-Length", strconv.FormatInt(contentLength, 10))
-	if pageBlobClearPagesOptions != nil && pageBlobClearPagesOptions.RangeParameter != nil {
-		req.Header.Set("x-ms-range", *pageBlobClearPagesOptions.RangeParameter)
+	if pageBlobClearPagesOptions != nil && pageBlobClearPagesOptions.RangeParam != nil {
```

Comment:
FYI that Parameter changed to Param.

Reply:
This is part of escaping keywords (I shortened it from "Parameter" to "Param").  However, in this case there's no need to escape it (I thought I had fixed this).  Will investigate.

Reply:
The suffix has been removed.